src/Jobs/Stages/PullFromSourceParallel.php

```diff
@@ -19,6 +19,11 @@ final class PullFromSourceParallel implements StageInterface
      */
     const DEFAULT_HANDLER_COUNT = 1;
 
+    /**
+     * @var string
+     */
+    const DEFAULT_QUEUE_NAME = 'indexing-queue';
```

Comment:
Better if less generic to prevent conflicts.

Reply:
changed to 'elasticsearch-parallel'